app/assets/stylesheets/index_page.scss

```diff
@@ -2,3 +2,21 @@
   margin-top: 20px;
   padding-left: 15px;
 }
+
+#indexMap {
```

Comment:
Avoid using id selectors<br>Selector `indexMap` should be written in lowercase with hyphens